Cargo.toml

```diff
@@ -48,6 +48,7 @@ harness = false
 concurrent = ["dep:rayon"]
 default = ["std", "concurrent"]
 executable = ["dep:clap", "dep:rand-utils", "std"]
+smt_hashmaps = []
```

Comment:
This was my first solution, but later I decided that we should use hashbrown hasmaps in other cases as well and included them unconditionally. I have no strong opinion regarding this particular case and if you think that it could be better to add the condition now and remove it later, I will update the feature.

Reply:
I would include `hashbrown` conditionally as I don't think we use it for anything else. If we do need it in the future, I think we can re-visit the approach.